docs/api/system-preferences.md

```diff
@@ -106,6 +106,15 @@ This is necessary for events such as `NSUserDefaultsDidChangeNotification`.
 
 Same as `unsubscribeNotification`, but removes the subscriber from `NSNotificationCenter`.
 
+### `systemPreferences.registerDefaults(defaults)` _macOS_
+
+* `defaults` Object - a dictionary of (`key: value`) user defaults
+  * `key` String
+  * `value` (String | Boolean | Number) - object of type `string`, `boolean`, `integer`, `float`, `double`,
+    `url`, `array`, or `dictionary`.
```

Comment:
Description of the `value` still contradicts value's type =/
Maybe it would be enough to keep only a formal type `(String | Boolean | Number)` and leave the description blank?

Reply:
sure, i think that's fine!

Reply:
> use floats, doubles, arrays, and dictionaries

It's a JavaScript API, and there is no such types in JavaScript as `float`, `double`, or `dictionary`, so it doesn't make sense to mention them in the docs.
But we could add `Array` and `Object` to the list of possible types.

Reply:
added them in!